wrapper/libsrc/combo_lib.c

```diff
@@ -0,0 +1,403 @@
+/**
```

Comment:
Not sure if we need to copy-paste a whole bunch of code just so that the calling interface can be changed. The typical way to do this is to create a new interface and link to the older one internally without re-writing code.